src/options.c

```diff
@@ -60,6 +60,7 @@ static char *costs_str;
 
 static struct opt_entry opt_list[] = {
 	{"", FLG_PASSWD, 0, 0, 0, OPT_FMT_ADD_LIST, &options.passwd},
+	{"help", FLG_NONE, 0, 0, 0, 0, 0},
```

Comment:
While this doesn't hurt, it's not the correct fix. We handle --help separately. The problem you were trying to solve is you still don't `exit(0)` in the right places.